File: stockfishlib/src/main/java/com/loloof64/android/stockfishlib/NativeLib.java
package com.loloof64.android.stockfishlib;

class NativeLib {
        // Used to load the 'stockfishlib' library on application startup.
        static {
            System.loadLibrary("stockfishlib");
        }

        public static native String readStdOut();

        public static native boolean writeStdIn(String command);

        public static native void init();

        public static native void main();
}

File: stockfishlib/src/main/java/com/loloof64/android/stockfishlib/NativeLib.kt
package com.loloof64.android.stockfishlib

class NativeLib {

    companion object {
        // Used to load the 'stockfishlib' library on application startup.
        init {
            System.loadLibrary("stockfishlib")
        }

        external fun readStdOut(): String

        external fun writeStdIn(command: String): Boolean

        external fun init()

        external fun main()
    }
}